TestSupport/planarity_orchestrator.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+__all__ = []
+
+import multiprocessing
+import subprocess
+import argparse
+from pathlib import Path
+
+_planarity_algorithm_commands =  ('p', 'd', 'o', '2', '3', '4')
+
+
+def call_planarity(
+        planarity_path:str, command:str, order:int, num_edges:int,
+        input_dir:Path, output_dir:Path):
+    """Call planarity as blocking process on multiprocessing thread
+
+    Uses subprocess.run() to start a blocking process on the multiprocessing
+    pool thread to call the planarity executable so that it applies the
+    algorithm specified by the command to all graphs in the input file and
+    outputs results to the output file, i.e. calls
+       planarity -t {command} {infile_path} {outfile_path}
+
+    Args:
+        planarity_path: Path to the nauty geng executable
+        command: Algorithm specifier character
+        order: Desired number of vertices
+        num_edges: Desired number of edges
+        input_dir: Directory containing the .g6 file with all graphs of the
+            given order and num_edges
+        output_dir: Directory to which you wish to write the output of applying
+            the algorithm corresponding to the command to all graphs in the
+            input .g6 file
+    """
+    infile_path = Path.joinpath(input_dir, f'n{order}.m{num_edges}.g6')
+    outfile_path = Path.joinpath(
+        output_dir, f'{command}', f'n{order}.m{num_edges}.{command}.out.txt')
+    subprocess.run(
+        [
+            f'{planarity_path}',
+            '-t', f'-{command}',
+            f'{infile_path}',
+            f'{outfile_path}'
+        ],
+        stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
```

Comment:
Same as in [this comment](https://github.com/graph-algorithms/edge-addition-planarity-suite/pull/74/files#r1614178925)